src/Utility/Language.php

```diff
@@ -14,15 +14,16 @@ class Language {
 	 */
 	public static function parseLanguageList($languageList = null) {
 		if ($languageList === null) {
-			if (empty($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
+			if (empty(env('HTTP_ACCEPT_LANGUAGE'))) {
```

Comment:
Well I am using it locally...  aka $preferredLocales = env('HTTP_ACCEPT_LANGUAGE'); is fetching it.
core's `function env($key, $default = null)` doesn't do it.